client/src/components/timeline/date-list.js

```diff
@@ -13,10 +13,10 @@ const DateList = ( { revisions } ) => {
 		prev = list;
 		return (
 			<div className="row day border pl-2 pr-2 pt-2 pb-2 mb-3" key={date.format( 'YYYY-MM-DD' )}>
-				<div className="col-xl-1 col-sm-2 col-12">
-					<Date icon="today" date={date.format( 'YYYY-MM-DD' )} />
+				<div className="col-xl-10 offset-xl-1 col-sm-8 col-12">
+					<Date icon="today" date={date.format( 'D MMMM Y' )} />
```

Comment:
Will this format make sense in all regions?

Reply:
Nope. Assuming this is using momentjs, then I recommend `LL` that picks an appropriate format for the locale that displays day, month and year.

Reply:
We originally had it that way, but it was changed in https://phabricator.wikimedia.org/T180725

Reply:
Given that now the date has as much space as it needs would it make sense to switch back to `LL`?
<img width="1135" alt="screen shot 2018-05-14 at 11 01 46 am" src="https://user-images.githubusercontent.com/9491/39979826-491c61b4-5766-11e8-9349-00148b8373d4.png">

Reply:
Do we have a good way for moment to be indicated the user's time prefs? I think moment does that on its own, but we do some magic in MediaWiki itself to push the chosen interface language.

In any case, I'm all for making the date more suitable for the user, so if LL works in moment, I'd say go with that.

Reply:
I think localizing the date format is the _proper_ thing to do, but again, this was a product decision, so a new ticket/discussion would have to happen (and ping Trevor). (or repoen https://phabricator.wikimedia.org/T180725)